Argo/Types/Decoded/Monad.swift

```diff
@@ -13,6 +13,21 @@ public func >>- <T, U>(x: Decoded<T>, @noescape f: T -> Decoded<U>) -> Decoded<U
   return x.flatMap(f)
 }
 
+/**
+  flatMap a function over a `Decoded` value (left associative)
+
+  - If the value is a failing case (`.TypeMismatch`, `.MissingKey`), the function will not be evaluated and this will return the value
+  - If the value is `.Success`, the function will be applied to the unwrapped value
+
+  - parameter x: A value of type `Decoded<T>`
+  - parameter f: A transformation function from type `T` to type `Decoded<U>`
+
+  - returns: A value of type `Decoded<U>`
+*/
+public func -<< <T, U>(@noescape f: T -> Decoded<U>, x: Decoded<T>) -> Decoded<U> {
```

Comment:
Valid Docs Violation: Documented declarations should be valid. (valid_docs)<br>Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)